src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/entities/WorkType.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.cft.entities;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.io.Serializable;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+
+@EqualsAndHashCode
+@Getter
+@ToString
+@Entity(name = "work_types")
+public class WorkType implements Serializable {
```

Comment:
I think the naming convention used by @indvamshi for entities was `<EntityName>Resource`.  Better if we follow that e.g.`WorkTypeResource`

Reply:
Done